File: seon7129/week4/1929.c
#include <stdio.h>
#include <math.h>

int main()
{
    int m, n;
    int i;
    int j;
    int flag = 0;

    scanf("%d", &m);
    scanf("%d", &n);

    i = m;
    if (i < 2)
        i = 2;
    while (i <= n)
    {
        j = 2;
        while (j <= sqrt(i))
        {
            if (i % j == 0)
            {
                flag = 1;
                break;
            }
            j++;
        }
        if(flag == 0)
            printf("%d\n", i);
        flag = 0;
        i++;
    }

    return (0);
}

File: gy0-0o/week4/1929.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		int M = scan.nextInt();
		int N = scan.nextInt();
		find_prime(M,N);
		scan.close();
	}
	
	public static void find_prime(int M, int N) {
		for (int i = M; i <= N; i++) {
			if (i < 2) {
				continue;
			}
			
			if (i == 2) {
				System.out.println("2");
				continue;
			}
			
			if (check(i) == 1) {
				continue;
			}
			System.out.println(i);
		}
	}
	
	public static int check(int i) {
		for (int num = 2; num <= Math.sqrt(i); num++) {
			if (i % num == 0) {
				return (1);
			}
		}
		return (0);
	}
}
